src/javascript/app/pages/user/new_account/welcome_page.js

```diff
@@ -1,20 +1,22 @@
-const BinarySocket   = require('../../../base/socket');
-const Client         = require('../../../base/client');
-const localize       = require('../../../../_common/localize').localize;
-const createElement  = require('../../../../_common/utility').createElement;
-const getElementById = require('../../../../_common/common_functions').getElementById;
-const Url            = require('../../../../_common/url');
+const BinarySocket      = require('../../../base/socket');
+const Client            = require('../../../base/client');
+const localize          = require('../../../../_common/localize').localize;
+const createElement     = require('../../../../_common/utility').createElement;
+const getElementById    = require('../../../../_common/common_functions').getElementById;
```

Comment:
@brandon-binary this import statement can be moved to line 3, before localize

Reply:
@ahkuma we sort import statements alphabetically, it should be like this
```
const Client                        = require('../../../base/client');
const BinarySocket            = require('../../../base/socket');
const getElementById        = require('../../../../_common/common_functions').getElementById;
const localize                     = require('../../../../_common/localize').localize;
const Url                            = require('../../../../_common/url');
const createElement         = require('../../../../_common/utility').createElement;
const showLoadingImage = require('../../../../_common/utility').showLoadingImage;
```

Reply:
@nazaninreihani Shouldn't ```BinarySocket``` comes after ```getElementById``` instead because ```s``` comes after ```c```?